config/redis.go

```diff
@@ -0,0 +1,63 @@
+package config
+
+import (
+	"regexp"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Redis configuration for the redis connection
+type Redis struct {
+	Address            string   `yaml:"address"`
+	Username           string   `yaml:"username" default:""`
+	Password           string   `yaml:"password" default:""`
+	Database           int      `yaml:"database" default:"0"`
+	Required           bool     `yaml:"required" default:"false"`
+	ConnectionAttempts int      `yaml:"connectionAttempts" default:"3"`
+	ConnectionCooldown Duration `yaml:"connectionCooldown" default:"1s"`
+	SentinelUsername   string   `yaml:"sentinelUsername" default:""`
+	SentinelPassword   string   `yaml:"sentinelPassword" default:""`
+	SentinelAddresses  []string `yaml:"sentinelAddresses"`
+}
+
+// IsEnabled implements `config.Configurable`
+func (c *Redis) IsEnabled() bool {
+	return c.Address != ""
+}
+
+// LogConfig implements `config.Configurable`
+func (c *Redis) LogConfig(logger *logrus.Entry) {
+	if len(c.SentinelAddresses) == 0 {
+		logger.Info("Address: ", c.Address)
+	}
+
+	logger.Info("Username: ", c.Username)
+	logger.Info("Password: ", obfuscatePassword(c.Password))
+	logger.Info("Database: ", c.Database)
+	logger.Info("Required: ", c.Required)
+	logger.Info("ConnectionAttempts: ", c.ConnectionAttempts)
+	logger.Info("ConnectionCooldown: ", c.ConnectionCooldown)
+
+	if len(c.SentinelAddresses) > 0 {
+		logger.Info("Sentinel:")
+		logger.Info("  MasterName: ", c.Address)
+		logger.Info("  Username: ", c.SentinelUsername)
+		logger.Info("  Password: ", obfuscatePassword(c.SentinelPassword))
+		logger.Info("  Addresses:")
```

Comment:
The other code mostly uses lower-case, and it has the benefit of mostly matching the option names:
```suggestion
		logger.Info("address: ", c.Address)
	}

	logger.Info("username: ", c.Username)
	logger.Info("password: ", obfuscatePassword(c.Password))
	logger.Info("database: ", c.Database)
	logger.Info("required: ", c.Required)
	logger.Info("connectionAttempts: ", c.ConnectionAttempts)
	logger.Info("connectionCooldown: ", c.ConnectionCooldown)

	if len(c.SentinelAddresses) > 0 {
		logger.Info("sentinel:")
		logger.Info("  master: ", c.Address)
		logger.Info("  username: ", c.SentinelUsername)
		logger.Info("  password: ", obfuscatePassword(c.SentinelPassword))
		logger.Info("  addresses:")
```